src/http-clients.md

```diff
@@ -1,4 +1,45 @@
 ---
 ---
 
-# Clients HTTP Interface
\ No newline at end of file
+# Clients HTTP Interface
+
+Details of a clients which will be accessing Learning Locker. It contains details for permissions, authenticating and storing the xapi request.
+
+It is accessible through the [connection API](/http-connection) and [restify api](/http-models).
```

Comment:
Can we change "connection API" to "Connection HTTP Interface" and "restify api" to "Models HTTP Interface" for consistency with the sidebar?

Reply:
Done.